src/workers.ts

```diff
@@ -27,7 +27,8 @@ const DEFAULT_WORKERS = {
   StreamSaver: 'streamsaver.penumbra.serviceworker.js',
 };
 
-const SHOULD_LOG_EVENTS = process.env.PENUMBRA_LOG_START === 'true';
+const SHOULD_LOG_EVENTS =
+  'process' in self && process.env.PENUMBRA_LOG_START === 'true';
```

Comment:
```suggestion
  typeof process !== 'undefined' && process.env.PENUMBRA_LOG_START === 'true';
```

slightly more correct (e.g. `globalThis.process` can be different than `self.process`)

Reply:
SG! Also, on the right-hand side, should it be `globalThis.process.env.PENUMBRA_LOG_START === 'true';`?

Reply:
FWIW this was causing some issues because the global types for `node` weren't added in tsconfig. I feel like that's reasonable to omit, but we'd have to update a few things - like this and all the refs to the `Buffer` type (or we could `import type { Buffer } from 'buffer' // ferross package`).

In any case, does this file execute in Node? I was confused when I saw it here

Reply:
This file executes in the browser. esbuild can inject variables like `process.env.*` through its `define` API. Here's an example of that: https://github.com/transcend-io/consent-manager-ui/blob/main/src/scripts/build.ts#L70-L75

Reply:
doing some of that in #229 - i think that branch is better than this one so we could close this out